packages/e2e-test-app/windows/ReactUWPTestApp/MainPage.xaml

```diff
@@ -8,6 +8,10 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-    <react:ReactRootView x:Name="myRootView" ComponentName="ReactUWPTestApp"
-        Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Height="600"/>
+    <react:ReactRootView 
+        x:Name="myRootView"
+        ComponentName="ReactUWPTestApp"
+        Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+        Width="800"
```

Comment:
Not sure if this is the default width, but CI should tell me 🤷‍♀️

Reply:
Snapshots match, so I think this was right.

Reply:
1074 is what we've needed so far, e.g. the display:none PR

Reply:
I looked through some snapshots and noticed that width, but also a lot of 800 width outer views.

IIRC my guess was that some views may overflow or be horizontally scrollable? 

I previously verified changing root view height affect tree dumps, so my assumption would be that width should have a similar effect. I didn't validate that a template uwp app has an 800x600 window size, but that number would make sense, and seems to lead to consistent snapshots.

Reply:
Confirmed 800x600 in code https://github.com/microsoft/react-native-windows/blob/6d9a804a6782b40dfeee5de0bfc1bfe91ab962dd/packages/e2e-test-app/windows/ReactUWPTestApp/App.xaml.cs#L77

Reply:
Though that does make me wonder if she issue Agne was seeing in https://github.com/microsoft/react-native-windows/issues/7589 is the same as the issue this solves. Might be good to confirm.